.github/workflows/conbench_upload.yml

```diff
@@ -0,0 +1,104 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Upload Benchmark Results
+on:
+  workflow_dispatch:
```

Comment:
I was gonna say we should delete this — if someone kicks it off accidentally, we don't want the duplicate to be uploaded — but the chance of that seems small, and it would be nice to be able to try uploading again if the Conbench server was down or something. So I like this.

Reply:
This was mainly for my own development so I don't have to wait 1h+ when stamping out typos and such :D

We can always remove it but as you need to be a member/collaborator to trigger the job the utility outweighs the potential for abuse imo.

Reply:
Dev/Testing a GHA workflow is an incredibly intense experience.